src/main/java/com/lorepo/icplayer/client/module/text/TextPresenter.java

```diff
@@ -1026,6 +1027,34 @@ protected void insertToGap(String gapId) {
 		}
 	}
 
+	private String getDraggableValue() {
+		String value = draggableItem.getValue();
+		if (hasTextStyle(value) || isMathFormula(value)) {
+			return value;
+		}
+		
+		return TextParser.removeHtmlFormatting(draggableItem.getValue());
+	}
+
+	private boolean hasTextStyle(String value) {
+		return value.contains("<b>") || value.contains("<i>") || value.contains("<s>");
+	}
+
+	private boolean isMathFormula(String value) {
+		String pattern = "[a-zA-Z0-9]+<[a-zA-Z0-9\\s]+.*";
```

Comment:
Rozumiem sprawdzenie czy to matematyczna formuła ale nie rozumiem zastosowania tego konkretnego regexa.
Widzę, że regex znajdzie `1<1` lecz `1 < 1` już nie. Jeżeli ten regex nie ma za zadanie złapać dowolnej matematycznej formuły a jakiś inny cel to proszę o bardziej dopasowaną nazwę.
![image](https://github.com/user-attachments/assets/41b6d9e3-95f1-4759-854d-9b24ece611f2)
Na powyższym zdjęciu podawane jest `x<3` i błąd dalej się reprodukuje.